src/Output/Color.php

```diff
@@ -95,6 +95,22 @@ public function info(string $text, array $style = []): string
         return $this->line($text, ['fg' => static::BLUE] + $style);
     }
 
+    /**
+     * Returns the color code for a 256 background color.
+     */
+    public function bg256(int $code)
```

Comment:
missing return type `: string` at the end

Reply:
Updated!